builder/openstack/step_source_image_info.go

```diff
@@ -0,0 +1,96 @@
+package openstack
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/gophercloud/gophercloud/openstack/imageservice/v2/images"
+	"github.com/gophercloud/gophercloud/pagination"
+	"github.com/hashicorp/packer/helper/multistep"
+	"github.com/hashicorp/packer/packer"
+)
+
+type StepSourceImageInfo struct {
+	SourceImage     string
+	SourceImageName string
+	ImageFilters    ImageFilterOptions
+}
+
+type ImageFilterOptions struct {
+	Filters    map[string]interface{} `mapstructure:"filters"`
+	MostRecent bool                   `mapstructure:"most_recent"`
+}
+
+func (s *StepSourceImageInfo) Run(_ context.Context, state multistep.StateBag) multistep.StepAction {
+	config := state.Get("config").(Config)
+	ui := state.Get("ui").(packer.Ui)
+
+	client, err := config.computeV2Client()
```

Comment:
Should be `imageV2Client`. When changing this the timestamps and other information about the image is returned correctly.

Reply:
Possible fix:
```
client, err := config.imageV2Client()
if err != nil {
	err := fmt.Errorf("Error getting image client: %s", err)
	state.Put("error", err)
	ui.Error(err.Error())
	return multistep.ActionHalt
}

```

Reply:
Yeahh definitely using the wrong client. :+1: